packages/deployer/src/handlers/aws/index.ts

```diff
@@ -1,13 +1,12 @@
 import * as fs from 'fs';
 import * as path from 'path';
 import { config, handlers, logger, promiseUtils, providerState } from '@api3/node';
-import * as node from '@api3/node';
 
 const configFile = path.resolve(`${__dirname}/../../config-data/config.json`);
 const rawConfig = JSON.parse(fs.readFileSync(configFile, 'utf8'));
 const parsedConfig = config.parseConfig(rawConfig[0]);
 
-function encodeBody(data: node.WorkerResponse): string {
+function encodeBody(data: any): string {
```

Comment:
Maybe it's better to just use `JSON.stringify()` directly in the `testApi` handler and keep the type here?